src/pybind/mgr/dashboard_v2/frontend/src/app/shared/components/table/table-key-value.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input } from '@angular/core';
+
+/**
+ * Display the given data in a 2 column datatable. The left column
+ * shows the 'key' attribute, the right column the 'value' attribute.
+ * The datatable has the following characteristics:
+ * - No header and footer is displayed
```

Comment:
You can pass a bool to the data table to not show the header. For the footer you can add an input to have no limit on the table.